### PR TITLE
Change "Abort" button to say "Cancel"

### PR DESCRIPTION
**Pull request by strugee**

### CODE REVIEW

File: app/views/scenarios/_confirm_deletion_modal.html.erb [hunk 1]
```diff
@@ -36,7 +36,7 @@
           </div>
         </div>
         <div class="modal-footer">
-          <%= f.button 'Abort', class: 'btn btn-default', 'data-dismiss' => 'modal' %>
+          <%= f.button Cancel', class: 'btn btn-default', 'data-dismiss' => 'modal' %>
```
Comment (dsander) [inline app/views/scenarios/_confirm_deletion_modal.html.erb]: You are missing the opening quote here.

Comment (strugee) [inline app/views/scenarios/_confirm_deletion_modal.html.erb]: @dsander wow, nice catch. Just pushed a fix.

**Comment by dsander** [comment]
Thanks @strugee!

**Comment by strugee** [comment]
:thumbsup: my pleasure.

**Pull request merged**